app/javascript/pages/HomePage.tsx

```diff
@@ -33,14 +34,18 @@ const HomePage = (props: HomePageProps) => {
       </div>
       <Hero
         title={t("welcome.title")}
-        buttonTitle={t("welcome.seeRentalListings")}
+        backgroundImage={getAssetPath("bg@1200.jpg")}
```

Comment:
it looks like for the angular app we might change the resolution of the picture based on the resolution of the device.  Going from bg@400.jpg to bg@2400.jpg. Is that something we should consider here too? Or should we check in with Exygy to see if they intentionally dropped that?

Reply:
I saw that but didn't haven't put much thought into how we'd do it. It's not supported in bloom, I'm not sure if it was conscious or not but I'm pretty sure they're not doing anything smart on their end to scale it for us.

It looks okay to me at all resolutions, but we could make a ticket to make this responsive? I think we should look into how to get the screen size via react hooks if that's possible

Reply:
might be worth asking Jesse about it to see if it's something on their list. If he thinks it's OK then I'm OK with it too.

Reply:
@james-wills-sf it seems like Jesse's answer was fairly non-committal, but they could maybe make an issue in bloom for it? Should we follow up and make sure that ticket gets created?

Reply:
Yeah I'll make sure an issue gets created (or I'll just create one)

Reply:
Issue linked here: https://github.com/bloom-housing/bloom/issues/1303